README.md

```diff
@@ -1,2 +1,140 @@
+```
+
+#### Multiaddrs
+
+You can specify the libp2p rendezvous server listen and announce multiaddrs. This server is configured with [libp2p-tcp](https://github.com/libp2p/js-libp2p-tcp) and [libp2p-websockets](https://github.com/libp2p/js-libp2p-websockets) and addresses with this transports should be used. It can always be modified via the API.
+
+```sh
+libp2p-relay-server --peerId id.json --listenMultiaddrs '/ip4/127.0.0.1/tcp/15002/ws' '/ip4/127.0.0.1/tcp/8000' --announceMultiaddrs '/dns4/test.io/tcp/443/wss/p2p/12D3KooWAuEpJKhCAfNcHycKcZCv9Qy69utLAJ3MobjKpsoKbrGA' '/dns6/test.io/tcp/443/wss/p2p/12D3KooWAuEpJKhCAfNcHycKcZCv9Qy69utLAJ3MobjKpsoKbrGA'
+```
+
+By default it listens on `/ip4/127.0.0.1/tcp/15002/ws` and has no announce multiaddrs specified.
+
+#### Metrics
+
+Metrics are enabled by default on `/ip4/127.0.0.1/tcp/8003` via Prometheus. This address can also be modified with:
+
+```sh
+libp2p-relay-server --metricsMultiaddr '/ip4/127.0.0.1/tcp/8000'
+```
+
+Moreover, metrics can also be disabled with:
+
+```sh
+libp2p-relay-server --disableMetrics
+```
+
+#### Discovery
+
+A discovery module [libp2p/js-libp2p-pubsub-peer-discovery](https://github.com/libp2p/js-libp2p-pubsub-peer-discovery) is configured and enabled by default. It can be disabled with:
+
+```sh
+libp2p-relay-server --disablePubsubDiscovery
+```
+
+This discovery module uses `_peer-discovery._p2p._pubsub` as a topic for enabling peer discovery. However, other topics can be used with:
+
+```sh
+libp2p-relay-server --discoveryTopics '_peer-discovery._app1._pubsub' '_peer_discovery._app2._pubsub'
+```
+
+### Docker
+
+When running the relay server in Docker, you can configure the same parameters via environment variables, as follows:
+
+```sh
+PEER_ID='./id.json'
+LISTEN_MULTIADDRS='/ip4/127.0.0.1/tcp/15002/ws,/ip4/127.0.0.1/tcp/8000'
+ANNOUNCE_MULTIADDRS='/dns4/test.io/tcp/443/wss/p2p/12D3KooWAuEpJKhCAfNcHycKcZCv9Qy69utLAJ3MobjKpsoKbrGA,/dns6/test.io/tcp/443/wss/p2p/12D3KooWAuEpJKhCAfNcHycKcZCv9Qy69utLAJ3MobjKpsoKbrGA'
+METRICS_MULTIADDR='/ip4/127.0.0.1/tcp/8000'
+DISABLE_METRICS='true'
+DISABLE_PUBSUB_DISCOVERY='true'
+DISCOVERY_TOPICS='_peer-discovery._app1._pubsub,_peer_discovery._app2._pubsub'
+```
+
+Please note that you should expose expose the used ports with the docker run command. The default ports used are `8003` for the metrics and `150003` for the websockets listener
+
+```sh
+docker build NAME -t libp2p-relay
+docker run -p 8003:8003 -p 15002:15002 -p 8000:8000 -e LISTEN_MULTIADDRS='/ip4/127.0.0.1/tcp/15002/ws,/ip4/127.0.0.1/tcp/8000' -d libp2p-relay
+```
+
+### SSL
+
+You should setup an SSL certificate with nginx and proxy to the API. You can use a service that already offers an SSL certificate with the server and configure nginx, or you can create valid certificates with for example [Letsencrypt](https://certbot.eff.org/lets-encrypt/osx-nginx). Letsencrypt won’t give you a cert for an IP address (yet) so you need to connect via SSL to a domain name.
```

Comment:
Not needed right now, but it would be cool to see an example dockerfile extending this one, templated with nginx and letsencrypt.

Reply:
Yes, this would be good! I added it in the libp2p production guides README https://github.com/libp2p/js-libp2p/pull/804#discussion_r533421193 as it should probably be not specific to the relay. We can also link it from here afterwards.